src/lib/plugin.js

```diff
@@ -234,7 +234,7 @@ module.exports = class PluginPaymentChannel extends EventEmitter2 {
         )
         const resp = protocolDataToIlpAndCustom(btpResponse)
         this._info = (resp.protocolMap && resp.protocolMap.info) || {}
-        this._prefix = this.getInfo().prefix
+        this._prefix = this._info.prefix
```

Comment:
If `this._prefix` is already defined, it should make sure `this._info.prefix` is equal and throw an error otherwise